src/foam/nanos/session/Session.js

```diff
@@ -14,6 +14,11 @@ foam.CLASS({
     'foam.nanos.auth.CreatedByAware'
   ],
 
+  imports: [
```

Comment:
I guess it doesn't hurt, but are these checks still needed?

Reply:
I'm not sure what you mean by "checks" here since you commented on the imports. Can you clarify?

If you mean the `authorizeOnXYZ` checks below, then yes, we do still need them because `AuthorizationDAO` calls the `authorizeOnXYZ` methods and we need a condition that will be true for superadmins and false for everyone else.

Reply:
Yes, that's what I meant. But only superadmins would even have access to the DAO now that we don't need to make the DAO directly available now that we have the service. It should only be superadmin and system accessing the DAO.

Reply:
Are you suggesting that we only need service-level permission checks then? Like `service.sessionDAO` and only give that to superadmins? We still want to serve it because we'd like to be able to manage sessions through the GUI.

Reply:
Resolved by 25d353a.